### DISCUSSION TITLE
Request: TF1 or TF1 4K (French TV)

### DISCUSSION THREAD

**Discussion by Baumhausbanane123e** [iptv-org/community#570] [Q&A] [answered]
Hello,
I'm desperately looking for a streaming link for french TV channel TF1. All the links/lists I tried didn't work. If it's important: I'm in Germany not in France!
If someone has a link for TF1 or even better the 4K Version TF1 4K I would greatly appreciate it!

Thanks!

**Comment by SebTV-prod** [comment, marked as answer]
There's no legal link to see TF1 outside France, because of DRM's and geoblock restrictions. You must get a French IP adress, as by a VPN, and therefore you'll have some links on GitHub or even more on My TF1+ platform, which one is totally free.
You have also swiss platforms online like Zattoo and Teleboy who can authorize to see TF1, and also M6, C8, TMC, TFX, NRJ 12, BFM TV, and many other French TNT channels (with swiss ad-breaks) from Germany, and there again it's totally free and legal.

**Reply by Baumhausbanane123e** [reply]
I wasn't necessary looking for a legal way... Amyways, I already have a legal platform to watch TF1 and other channels. I was just looking for a direct m3u link to add to my media server.

> There's no legal link to see TF1 outside France, because of DRM's and geoblock restrictions. You must get a French IP adress, as by a VPN, and therefore you'll have some links on GitHub or even more on My TF1+ platform, which one is totally free. You have also swiss platforms online like Zattoo and Teleboy who can authorize to see TF1, and also M6, C8, TMC, TFX, NRJ 12, BFM TV, and many other French TNT channels (with swiss ad-breaks) from Germany, and there again it's totally free and legal.

**Comment by SebTV-prod** [comment]
And so, you can't found something illegal here because of DMCA. Maybe some
users could help you, but it'll be confidential and not by there, publicly.

Good luck ;)

**Reply by dkrtty** [reply]
Bonjour je peut avoir un lien mu3 pour regarder l'euro? Svp...

**Comment by dkrtty** [comment]
Bonjour je peut avoir un lien mu3 pour regarder l'euro? Svp...

**Reply by SebTV-prod** [reply]
Bonjour, non, les chaînes sportives illégales ne sont pas disponibles sur ce site. Cordialement.

**Reply by Baumhausbanane123e** [reply]
> Bonjour je peut avoir un lien mu3 pour regarder l'euro? Svp...

La platform Molotv TV doit être gratuite si je em rapelle bien. Il y a TF1 la.
